polynote-frontend/polynote/cell.js

```diff
@@ -1,5 +1,7 @@
 import {blockquote, div, iconButton, span, tag, button} from "./tags.js";
 import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
+import {StandardKeyboardEvent} from 'monaco-editor/esm/vs/base/browser/keyboardEvent.js'
```

Comment:
So this directly imports stuff from the Monaco code itself, rather than strictly using APIs. Not sure how safe this is...